static/src/javascripts/es6/bootstraps/app.js

```diff
@@ -0,0 +1,14 @@
+import crosswords from './crosswords';
+import crosswordThumbnails from 'es6/projects/common/modules/crosswords/thumbnails';
+
+var boot = function () {
+    crosswords.init();
+    crosswordThumbnails.init();
+};
+
+// TODO: IE8
```

Comment:
I only learnt this fact recently.

We don't yet apply the "cutting the mustard" technique in the jspm app. Can you advise?